src/main/scala/org/apache/spark/rdd/OrderedLeftJoinRDD.scala

```diff
@@ -0,0 +1,44 @@
+package org.apache.spark.rdd
+
+import org.apache.spark._
+import org.broadinstitute.hail.Utils._
+
+import scala.reflect.ClassTag
+
+class OrderedLeftJoinRDD[T, K, V1, V2](rdd1: OrderedRDD[T, K, V1], rdd2: OrderedRDD[T, K, V2])
```

Comment:
Ins't this the same as `OrderedLeftJoinTransformedRDD` with `K1 = K2`?  Let's delete this and call that `OrderedLeftJoinRDD`?